app/cli/index.js

```diff
@@ -82,15 +88,18 @@ const lsRemote = pattern => {
         entry.name = chalk.green(entry.name);
         return entry;
       })
-    );
+    )
+    .finally(() => process.exit(0));
 };
 
 args.command(['s', 'search'], 'Search for plugins on npm', (name, args_) => {
+  commandInvoked = true;
   const spinner = ora('Searching').start();
   const query = args_[0] ? args_[0].toLowerCase() : '';
-
-  commandPromise = lsRemote(query)
+  console.log(1);
```

Comment:
probably don't need this here?

Reply:
Yup good catch! There's also a separate log statement on 102 we can delete